src/types/lib/types.vcxproj.filters

```diff
@@ -1,27 +1,115 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
-    <Natvis Include="$(SolutionDir)tools\ConsoleTypes.natvis" />
+    <Filter Include="Source Files">

```

Comment:
I swear #1691 has the _opposite_ change in it...

Reply:
Yeah. TBH, I think this came in from a bad merge so I flipped a coin and picked one... I'll swap it, I guess?

Reply:
Fixed and I pushed the same file to the base branch too.